cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -499,3 +499,454 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
 
     # Clean up if it appeared
     snapshot.delete()
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_rhos_notification_for_snapshot_failures():
+    """
+    Test if cfme can report failure when deleting or creating
+    snapshot on RHOS.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
+        caseimportance: medium
+        initialEstimate: 1/4h
+        title: RHOS: test notification for snapshot create/delete failure
+        setup:
+            1. Add RHOS provider
+            2. Create a test VM on RHOS provider
+        testSteps:
+            1. Try to introduce a failure while creating snapshot
+                * Delete the VM in the process of creating a snapshot
+            2. Try to introduce a failure while deleting snapshot
+                * Disconnect machine from network and then try to delete snapshot
+        expectedResults:
+            1. Notification for failure when creating the snapshot appears
+            2. Notification for failure when deleting the snapshot appears
+    Bugzilla:
+        1581793
+
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.snapshot
+@pytest.mark.tier(1)
+def test_notification_for_snapshot_actions_on_openstack():
+    """
+    Test task notification for snapshot tasks: success of create
+    and delete snapshot.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Cloud
+        caseimportance: medium
+        initialEstimate: 1/3h
+        title: Test notification for snapshot actions on OpenStack
+        setup:
+            1. Add RHOS provider
+            2. Create a test VM on RHOS provider
+        testSteps:
+            1. Create a snapshot
+            2. Delete the snapshot
+        expectedResults:
+            1. Snapshot created successfully; notification displayed
+            2. Snapshot deleted successfully; notification displayed
+    Bugzilla:
+        1429313
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_sui_create_snapshot_from_vm_details_page_snapshot_page_and_service_details_page():
```

Comment:
Let's try to shorten this test name please.

Reply:
Done.